js/main.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener("DOMContentLoaded", function() {
+  fillRegionSelect(regions);
+});
+document.querySelector("form").addEventListener("change", validate);
+const submit = document.querySelector('#submit');
+const checkbox = document.querySelector('#checkbox');
+const option = (...args) => new Option(...args);
+
+function validate(ev) {
```

Comment:
updateForm, or renderForm would be a better name. You are manipulation the DOM, so it's not pure validation.